frontends/infinite-corridor/src/scss/button.scss

```diff
@@ -0,0 +1,20 @@
+@use "sass:color";
+
+.ol-btn {
```

Comment:
For the moment this button style (and `one-column`) below are mostly copied from open-discussions.

I was initially reluctant to move lots of styling to shared areas because I wanted to include only what we actually needed in infinite-corridor... I'd like to work on that a bit tomorrow (in a different PR), but IMO we should review this PR as-is for now, since I think it has stuff that will be helpful to other people.